library/src/test/java/com/yayandroid/locationmanager/helper/LogUtilsTest.java

```diff
@@ -0,0 +1,82 @@
+package com.yayandroid.locationmanager.helper;
+
+import com.yayandroid.locationmanager.helper.logging.Logger;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.internal.verification.VerificationModeFactory.times;
+
+public class LogUtilsTest {
+
+    @Mock
+    private Logger mockLogger;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+        LogUtils.setLogger(mockLogger);
+    }
+
+    @Test
+    public void whenLoggingIsDisabledItShouldNotForwardToLogger() {
+        LogUtils.enable(false);
+
+        LogUtils.logD("Dmessage");
+        LogUtils.logE("Emessage");
+        LogUtils.logI("Imessage");
+        LogUtils.logV("Vmessage");
+        LogUtils.logW("Wmessage");
+
+        verify(mockLogger, times(0)).logD(anyString(), anyString());
```

Comment:
You could also use `verifyZeroInteractions` here

Reply:
Nice! :heavy_check_mark: